### DISCUSSION TITLE
OBS Studio 27.1 Release Candidate 1, OBS becomes streamlabs?

### DISCUSSION THREAD

**Discussion by ogmkp** [obsproject/obs-studio#5196] [General]
```
Added YouTube integration [yuriy-chumak]
Allows the ability to connect your YouTube account without using a stream key
Adds various features when starting your stream that allows you to set settings for each stream: your stream title, description, privacy settings, scheduling streams, and more
Also features a chat dock for public and unlisted broadcasts (currently read-only) [derrod]
Special thanks to the developers at YouTube for working with us directly on the initial implementation of this feature and helping to make it a reality

```

That's nice but I'm not interested.
This should be a plugin as suggested in the project discussions.
If I want to manage and mix up everything, video production and management of Youtube with a chat and others smileys, I pick streamlabs!

**Comment by sebastian-s-beckmann** [comment]
You will still be able to use a stream key.

**Comment by derrod** [comment]
Right now this kind of integration is not really possible via plugin. If you don't like it just don't use it.

**Comment by mihawk90** [comment]
I'm wondering what happens when OP finds out there has been Twitch and Restream integration for well over a year too :see_no_evil:

**Reply by ogmkp** [reply]
This is only a listing of servers, not an account management.

**Reply by sebastian-s-beckmann** [reply]
> This is only a listing of servers, not an account management.

Wrong.
<img width="1639" alt="Bildschirmfoto 2021-08-26 um 19 57 13" src="https://user-images.githubusercontent.com/59806498/131012504-7b1e66f7-6a77-4b97-a038-202ad8d1a545.png">

**Reply by ogmkp** [reply]
I don't have this on Linux for 27.0.1.

![Capture du 2021-08-26 20-00-01](https://user-images.githubusercontent.com/17526713/131012855-69a750a4-b853-4902-be0f-8f43290bb820.png)

"Get stream key" opens the twitch website in my browser (externally)

**Reply by mihawk90** [reply]
> I don't have this on Linux for 27.0.1.

That depends entirely on the distro you're using.
Officially only the Ubuntu PPA has the integration.

[My Fedora RPM](https://github.com/mihawk90/obs-studio.spec) has it, but that's not official and I'm not aware of any other community-made packages that do.

**Reply by tt2468** [reply]
> I don't have this on Linux for 27.0.1.

Because the integration is not available on Linux. I'm not sure the YouTube integration is available for Linux either.

**Reply by mihawk90** [reply]
> > I don't have this on Linux for 27.0.1.
> 
> Because the integration is not available on Linux. I'm not sure the YouTube integration is available for Linux either.

Not true.
https://github.com/obsproject/obs-studio/releases/tag/27.0.0

> New Features and Additions
> Added service integration and browser dock support to macOS and Linux [tbodt, cg2121]

But as already stated above, officially this is only available on Ubuntu using the PPA.
The reason being that no other distro maintainer has access to the required Client IDs. I'm not aware of any distro that builds with CEF in the first place, much less the service integration. And since they don't build with it, they have not requested the Client IDs (although I am unsure if it would be given out anyway).

**Comment by Fenrirthviti** [comment]
This integration is presented in the same way as the other service integrations we have. We do not required it, it's optional. If you don't want to use it, then don't use it.

Service integrations are only available when OBS is built with browser source/CEF. Looks like the package you're using wasn't built with browser source. You won't see YouTube either, assuming the version you install was not compiled with browser source.

**Reply by ogmkp** [reply]
Yes it is builded with browser source following the wiki:
https://obsproject.com/wiki/install-instructions#debian-based-build-directions

**Reply by Fenrirthviti** [reply]
Then no valid client IDs were provided as part of the build. Again, if you don't want to use it, just don't use it. We're not going to remove a feature that thousands of our users have been asking us directly for because of one person who thinks we shouldn't have it.

**Reply by ogmkp** [reply]
Anyway I don't pretend to have it removed, I just point out that I don't like it.
It starts with that, then there will be other features and we will end up with a streamlabs like who forked OBS because they don't have integrations originally 😉.
I would like OBS to remain a live video production software and not a bunch of stuff. 
But the work is notable.

**Reply by Fenrirthviti** [reply]
You seem to have a misconception on why Streamlabs forked our program, but thank you for the concern. Our utmost priority with the development of OBS is to empower our users to have the tools they need to produce content in a free and open way. We will never require any of these integrations, they will always remain optional. They are added as a convenience for those who wish to use them. Our users always come first.
